## <span style="color:darkblue"> Final Exam

Stat140-02: Introduction to the Ideas and Applications of Statistics 

Due: Monday, May 7 @12pm

*Under no circumstancs should a student discuss the content of this exam with other students or the TAs. All students are expected to follow the Honor Code: “I will honor myself, my fellow students, and Mount Holyoke College by acting responsibly, honestly, and respectfully in both my words and deeds.” Students are encouraged to speak with the instructor if unclear about what constitutes an Honor Code violation. Violations of the Honor Code may result in failure of this course.* 

#### NAME: 

**Problem 1 (30 points)**

In chronic diseases such as multiple sclerosis, medications are approved for sale after the US Food and Drug Administration evaluates both the effectiveness of the medication in delaying the progression of the disease and the extent to which the medication provides symptom relief. An earlier study showed that two treatments (labeled $A$ and $B$) were equally effective in delaying the progression of disease. This problem examines simulated data from a study comparing how well the treatments relieve disease symptoms.  

A total of 410 participants were randomly assigned to one of the two treatments. The effect of disease symptoms on daily living  was assessed by responses to the EuroQol 5D-5L (EQ-5D-5L), a questionnaire widely used to study health-related quality of life; higher scores are indicative of better quality of life. The EQ-5D-5L was administered to each participant before the start of treatment and again after 6 months of treatment.  Investigators planned to compare the mean change in the EQ-5D-5L score between the two groups in order to assess whether the extent of symptom relief differs by treatment.

The data from the study are contained in the file `quality_of_life.Rdata`.  The dataset `quality.of.life` contains the following variables:

  - `treatment.group`: The treatment to which the participant was assigned, coded \texttt{A} or \texttt{B}.

  - `pre.treatment.score`: The EQ-5D-5L score just before treatment started.

  - `post.treatment.score`: The EQ-5D-5L score after 6 months of treatment.

Do the data from this study provide definitive evidence that the two treatments are different in their ability to offer symptom relief?  

  a) Outline a plan for analyzing the data. Specify how the variables in the dataset will be used, in addition to any variables that might be created. Clearly indicate which methods will be used in the analysis. If a hypothesis test will be conducted, state the null and alternative hypotheses and use significance level $\alpha = 0.05$.

  b) Conduct the analysis and summarize the numerical results. Be sure to provide an answer to the original question in the context of the data.

In [1]:
load("quality_of_life.Rdata")
head(quality.of.life)

treatment.group pre.treatment.score post.treatment.score
1 B               70.7                64.0                
2 A               60.8                67.7                
3 A               67.5                64.9                
4 B               53.7                54.1                
5 B               48.8                53.6                
6 A               50.2                49.5

#### Problem 2 (40 points)

In class we have talked about using the BRFSS data to address the question "Do men and women think differently about their body weight?". In this problem we further explore this question.

  a) Draw a random sample of size 300 from the data, using 2017 in the `set.seed()` function. Name the sample `brfss.samp`.
    
  b) Fit a linear model to predict desired weight from actual weight and assess whether actual weight is a significant predictor of desired weight.
  
  c) Fit a linear model to predict desired weight from actual weight and sex. Compare this model to the one fitted in part b); state which model is preferable, and explain your reasoning.
  
  d) Investigate whether the association of desired weight with actual weight is different for males versus females.
  
- Create a plot that shows the regression line for desired weight versus actual weight in males and the regression line for desired weight versus actual weight in females.
  
- Create a model that includes actual weight, sex, and the interaction between actual weight and sex. Write the model equation.
  
- Write the prediction equation for males and and the prediction equation for females. 
      
- Is there statistically significant evidence of an interaction between actual weight and sex? Explain your answer.
      
      


In [2]:
load("BRFSS.Rdata")
head(BRFSS)

x.state age weight wtdesire height htf hti sex sex.factor height.total
106107 31      60  200    150      508    5    8  1   Female     68          
91707  27      25  145    125      506    5    6  1   Female     66          
144918 45      40  180    170      511    5   11  0   Male       71          
100346 29      53  210    175      511    5   11  0   Male       71          
15880   8      80  170    170      504    5    4  1   Female     64          
78461  24      71  108    108      501    5    1  1   Female     61          
       bmi     
106107 30.40657
91707  23.40106
144918 25.10216
100346 29.28586
15880  29.17725
78461  20.40419

#### Problem 3 (30 points)

Habitat fragmentation refers to the process by which a habitat in a large contiguous space is divided into smaller, more isolated pieces; human activities such as agricultural development can result in habitat fragmentation. Smaller patches of habitat are only able to support limited populations of organisms, which reduces genetic diversity and overall population fitness.

You have taken a post-doctoral fellowship with a research team that studies the abundance of forest birds in an area in Australia. Counting birds is a difficult, time-consuming process. The research team is interested in whether bird abundance can be reliably predicted from the measured features of a patch, and have collected data on bird abundance and various patch characteristics (area, altitude, distance to nearest patch, grazing intensity). 

You have agreed to be responsible for building a prediction model based on these data.

The file `forest_birds.Rdata` contains a simplified version of data from a study of 56 forest patches in Australia. The variables in `forest.birds` are:

  - `abundance`:  Average number of forest birds observed in the patch, as calculated from several independent 20-minute counting sessions. 

  - `log.area`: Natural log of the area of the patch. Areas were originally measured in hectares; 1 hectare is 10,000 square meters and approximately 2.47 acres.

  - `log.distance`: Natural log of the distance to the nearest patch, originally measured in kilometers.

  - `altitude`: altitude of the patch, measured in meters.

  - `grazing.intensity`: a score indicating the extent of livestock grazing. The categories are: light (`1`), less than average (`2`), average (`3`), moderately heavy (`4`), heavy (`5`).

In your report for the research team, present a model for predicting forest bird abundance. Briefly explain the work done to develop the model and evaluate the model's strengths and weaknesses. The team has also requested that you make a prediction for a patch that has the following features: area 10 hectares, altitude 150 meters, distance to nearest patch 200 km, and moderately heavy grazing intensity. Use only the information necessary to make a prediction from your model.

In [3]:
load("forest_birds.Rdata")
head(forest.birds)

abundance grazing.intensity log.area   log.distance altitude
1  5.3      2                 -2.3025851 3.663562     160     
2  2.0      5                 -0.6931472 5.455321      60     
3  1.5      5                 -0.6931472 4.644391     140     
4 17.1      3                  0.0000000 4.189655     160     
5 13.8      5                  0.0000000 5.505332     140     
6 14.1      3                  0.0000000 5.455321     130

In [4]:
forest.birds$grazing.factor = factor(forest.birds$grazing.intensity, 
                                     levels = c(1, 2, 3, 4, 5),
                                     labels = c("light", "less than average",
                                                "average", "moderately heavy",
                                                "heavy"))
head(forest.birds)

abundance grazing.intensity log.area   log.distance altitude
1  5.3      2                 -2.3025851 3.663562     160     
2  2.0      5                 -0.6931472 5.455321      60     
3  1.5      5                 -0.6931472 4.644391     140     
4 17.1      3                  0.0000000 4.189655     160     
5 13.8      5                  0.0000000 5.505332     140     
6 14.1      3                  0.0000000 5.455321     130     
  grazing.factor   
1 less than average
2 heavy            
3 heavy            
4 average          
5 heavy            
6 average